src/test_rig_bluesky/plans.py

```diff
@@ -42,3 +43,34 @@ def spectroscopy(
     spec = spec or Line(sample_stage.x, 0, 5, 5)
 
     yield from spec_scan({spectroscopy_detector, sample_stage}, spec, metadata=metadata)
+
+
+def demo_spectroscopy(
+    spectroscopy_detector: AravisDetector = spectroscopy_detector,
+    sample_stage: XYZStage = sample_stage,
+    numbers_of_points: int = 5,
```

Comment:
Isn't this too small for a default? rounds down to a 2x2 scan. Maybe 25 is a better guess

Reply:
good suggestion I will update it.